frontend/packages/ux-editor/src/components/config/FormComponentConfig.test.tsx

```diff
@@ -85,6 +42,70 @@ describe('FormComponentConfig', () => {
     });
   });
 
+  it('should render list of unsupported properties', async () => {
```

Comment:
I can't see any need for the async here? 😄 
```suggestion
  it('should render list of unsupported properties', () => {
```

Reply:
Looks like we can remove it from some of the others it's as well.